Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Thermodynamics"
 uuid = "b60c26fb-14c3-4610-9d3e-2d17fe7ff00c"
 authors = ["Climate Modeling Alliance"]
-version = "0.5.11"
+version = "0.6.0"
```

Comment:
I suppose the compat entry below should be `CLIMAParameters = "0.4"`

Reply:
done